checker/index.php

```diff
@@ -127,8 +127,8 @@
 if ($_POST["validate_uri"])
 {
 	$_POST['uri'] = htmlentities($_POST['uri']);
-	
-	$uri = Utility::getValidURI($addslashes($_POST["uri"]));
+	$url=$addslashes+'('+$db+','+$_POST["uri"]+')'; 
```

Comment:
Does this change work for you?

Reply:
sorry for this mistake ,it's updated in the new commit